configurator/backend/authorization/service.go

```diff
@@ -113,6 +114,7 @@ func CreateSSOProvider(vp *viper.Viper) SSOProvider {
 			Product:               vpSSO.GetString("product"),
 			Host:                  vpSSO.GetString("host"),
 			AccessTokenTTLSeconds: vpSSO.GetDuration("access_token_ttl_seconds"),
+			ConfiguratorRootPath:  vpSSO.GetString("configurator_root_path"),
```

Comment:
It feels like this should be a more general configuration parameter `ui.base_url`, like in server already. This would allow the configuration parameter to be repurposed (and it feels like we may want to use it elsewhere).